Unable to start activity

```
FATAL EXCEPTION: main
Process: io.treehouses.remote, PID: 26129
java.lang.RuntimeException: Unable to start activity ComponentInfo{io.treehouses.remote/io.treehouses.remote.InitialActivity}: kotlin.UninitializedPropertyAccessException: lateinit property mChatService has not been initialized
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4164)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4322)
at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2685)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loopOnce(Looper.java:230)
at android.os.Looper.loop(Looper.java:319)
at android.app.ActivityThread.main(ActivityThread.java:8893)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:608)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property mChatService has not been initialized
at io.treehouses.remote.BaseInitialActivity.getMChatService(BaseInitialActivity.kt:40)
at io.treehouses.remote.BaseInitialActivity.getChatService(BaseInitialActivity.kt:54)
at io.treehouses.remote.bases.BaseFragment.onAttach(BaseFragment.kt:25)
at io.treehouses.remote.ui.home.HomeFragment.onAttach(HomeFragment.kt:309)
at androidx.fragment.app.Fragment.performAttach(Fragment.java:3071)
at androidx.fragment.app.FragmentStateManager.attach(FragmentStateManager.java:502)
at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:271)
at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1663)
at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3247)
at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:3154)
at androidx.fragment.app.FragmentController.dispatchCreate(FragmentController.java:252)
at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:219)
at io.treehouses.remote.BaseInitialActivity.onCreate(BaseInitialActivity.kt:43)
at io.treehouses.remote.InitialActivity.onCreate(InitialActivity.kt:37)
at android.app.Activity.performCreate(Activity.java:8935)
at android.app.Activity.performCreate(Activity.java:8904)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4146)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4322) 
at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103) 
at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139) 
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96) 
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2685) 
at android.os.Handler.dispatchMessage(Handler.java:106) 
at android.os.Looper.loopOnce(Looper.java:230) 
at android.os.Looper.loop(Looper.java:319) 
at android.app.ActivityThread.main(ActivityThread.java:8893) 
at java.lang.reflect.Method.invoke(Native Method) 
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:608) 
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103) 
```

--- a/app/src/main/kotlin/io/treehouses/remote/bases/BaseFragment.kt
+++ b/app/src/main/kotlin/io/treehouses/remote/bases/BaseFragment.kt
@@ -22,20 +22,17 @@ open class BaseFragment : Fragment() {
     override fun onAttach(context: Context) {
         super.onAttach(context)
         listener = if (context is HomeInteractListener) context else throw RuntimeException("Implement interface first")
-        mChatService = listener.getChatService()
         mBluetoothAdapter = (context.getSystemService(Context.BLUETOOTH_SERVICE) as? BluetoothManager)?.adapter
     }
 
     fun isListenerInitialized() = ::listener.isInitialized
 
-
     open fun chatOpen():Boolean {
         return this::mChatService.isInitialized
     }
 
-
     protected fun onLoad(mHandler: Handler?) {
-//        mChatService = listener.getChatService()
+        mChatService = listener.getChatService()
         mChatService.updateHandler(mHandler!!)
         mBluetoothAdapter = (context?.getSystemService(Context.BLUETOOTH_SERVICE) as? BluetoothManager)?.adapter
 

